examples/DancingGoat/DataProtectionSamples/DancingGoatSamplesModule.cs

```diff
@@ -1,7 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-
-using CMS;
+﻿using CMS;
```

Comment:
I know it's out of the scope of this task, but can you please fix all warnings in this file as well? It doesn't seem like a lot of work.